RELEASE-NOTES.txt

```diff
@@ -1,6 +1,7 @@
 4.9
 -----
 - [*] Edit Products: the update action now shows up on the product details after updating just the sale price.
+- [*] Variable product > variation list: a warning banner is shown if any variations do not have a price, and warning text is shown on these variation rows.
```

Comment:
Should we remove this since this will be released as part of M3?

Reply:
The warning UI updates https://github.com/woocommerce/woocommerce-ios/issues/2594 aren't behind the M3 feature flag, and this variation list screen is available for old product details as well (tap on a variable product > variations row, just the cells aren't tappable). I think it could be helpful to include this in the release notes